app/routes.py

```diff
@@ -1,2 +1,296 @@
+            video.release_date = form_data["release_date"]
+            video.total_inventory = form_data["total_inventory"]
+            db.session.commit()
+            return video.to_json(), 200
+    
+    # DELETE a specific video
+    elif request.method == "DELETE":
+        db.session.delete(video)
+        db.session.commit()
+
+        return jsonify({
+            "id": video.id,
+            "details": f'Video {Video.title} successfully deleted'}), 200
+
+
+
+# <------------------------------------- RENTAL ENDPOINTS ------------------------------------->
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix= "/rentals")
+# POST /rentals/check-in -> checks out a video to a customer and updates the data in the db 
+@rentals_bp.route("/check-out", methods=["POST"], strict_slashes=False)
+def rental_check_out():
+    request_body = request.get_json()
+
+    customer_id = request_body.get("customer_id")
+    video_id = request_body.get("video_id")
+
+    if type(customer_id) != int or type(video_id) != int:
+        return make_response({"details": "Invalid data"}), 400
+    
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+
+    if customer is None and video is None:
+        return make_response(), 404
+
+    if video.available_inventory < 1:
+        return jsonify({"details": "Invalid data"}), 400
+
+    customer.videos_checked_out_count += 1
+    video.available_inventory -= 1
+
+    new_rental = Rental(
+        customer_id=customer_id,
+        video_id=video_id,
+        due_date=datetime.now() + timedelta(days=7)
+    )
+
+    db.session.add(new_rental)
+    db.session.commit()
+
+    return jsonify({
+        "customer_id": new_rental.customer_id,
+        "video_id": new_rental.video_id,
+        "due_date": new_rental.due_date,
+        "videos_checked_out_count": customer.videos_checked_out_count,
+        "available_inventory": video.available_inventory
+    }), 200
+
+# POST /rentals/check-in -> checks in a video to a customer and updates the data in the db
+@rentals_bp.route("/check-in", methods=["POST"], strict_slashes=False)
+def rental_check_in():
+    request_body = request.get_json()
+
+    customer_id = request_body.get("customer_id")
+    video_id = request_body.get("video_id")
+
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+
+    for rental in customer.video:
+        if rental.video_id == video_id:
+            customer.videos_checked_out_count -= 1
+            video.available_inventory += 1
+            db.session.delete(rental)
+            db.session.commit()
+        
+            return jsonify({
+                "customer_id": customer_id,
+                "video_id": video_id,
+                "videos_checked_out_count": customer.videos_checked_out_count,
+                "available_inventory": video.available_inventory
+            }), 200
+
+    else:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
For consistency, consider first checking for invalid data, and then getting to the main business of the method if that data is valid.